src/Resolvers/BrandResolver.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Rapidez\AmastyShopByBrand\Resolvers;
+
+use Illuminate\Support\Facades\DB;
+use Rapidez\Core\Facades\Rapidez;
+
+class BrandResolver
+{
+    public function byPath() : ?object
```

Comment:
I'd suggest adding the wanted path as a function parameter, it offers more flexibility if you want to get them on another page for some reason